site/snippets/turnkey.ts

```diff
@@ -1,47 +1,34 @@
-import { LocalAccountSigner, SmartAccountSigner } from "@alchemy/aa-core";
+import { TurnkeySigner } from "@alchemy/aa-signers";
 import { ApiKeyStamper } from "@turnkey/api-key-stamper";
-import { TurnkeyClient } from "@turnkey/http";
-import { createAccount } from "@turnkey/viem";
+import { http } from "viem";
 
-export async function newTurnkeySigner() {
-  const turnkeyClient = new TurnkeyClient(
-    {
-      /* 
-        Configurable, but you will likely use "https://api.turnkey.com".
-      */
-      baseUrl: process.env.TURNKEY_BASE_URL!,
-    },
-    new ApiKeyStamper({
-      /* 
-        You will generate these values as part of our quickstart guide: 
-        
-        https://docs.turnkey.com/getting-started/quickstart
-        
-        They are what the signer will use to authenticate requests to the Turnkey.com.
-      */
-      apiPublicKey: process.env.TURNKEY_API_PUBLIC_KEY!,
-      apiPrivateKey: process.env.TURNKEY_API_PRIVATE_KEY!,
-    })
-  );
+const TURNKEY_BASE_URL = "https://api.turnkey.com";
 
-  const turnkeyAccount = await createAccount({
-    client: turnkeyClient,
-    /* 
-        You can pull this from the top right widget in the Turnkey dashboard. 
-        The quickstart guide details how to do this.
-    */
-    organizationId: process.env.TURNKEY_ORGANIZATION_ID!,
-    /* 
-        This value should be the ID of the private key that you want to own
-        the light smart contract. Similarly pulled from the dashboard and
-        documented in the quickstart guide.
-    */
-    privateKeyId: process.env.TURNKEY_PRIVATE_KEY_ID!,
+// These are generated using the [Turnkey quickstart guide](https://docs.turnkey.com/getting-started/quickstart)
+// and can be found on the [Turnkey Dashboard](https://app.turnkey.com/dashboard/).
+const TURNKEY_API_PUBLIC_KEY = "test";
+const TURNKEY_API_PRIVATE_KEY = "test";
+const TURNKEY_ORGANIZATION_ID = "12345678-1234-1234-1234-123456789abc";
+
+// This is a wallet address or ID.
+const TURNKEY_SIGN_WITH = "0x1234567890123456789012345678901234567890";
+
+export const createTurnkeySigner = async () => {
+  const turnkeySigner = new TurnkeySigner({
+    apiUrl: TURNKEY_BASE_URL,
+    // API Key, WebAuthn, and Email Auth [stampers](https://docs.turnkey.com/category/api-design)
```

Comment:
nit: should this be:
```suggestion
    // API Key, WebAuthn, or Email Auth [stampers](https://docs.turnkey.com/category/api-design)
```